SPECS-EXTENDED/gstreamer1/gstreamer1.spec

```diff
@@ -72,51 +55,30 @@ Conflicts:      gstreamer1-plugins-bad-free-devel < 1.13
 The %{name}-devel package contains libraries and header files for
 developing applications that use %{name}.
 
-
-%package devel-docs
-Summary:         Developer documentation for GStreamer streaming media framework
-Requires:        %{name} = %{version}-%{release}
-BuildArch:       noarch
-
-%description devel-docs
-This %{name}-devel-docs contains developer documentation for the
-GStreamer streaming media framework.
-
 %prep
 %setup -q -n gstreamer-%{version}
 %patch0 -p1 -b .rpm-provides
 
 %build
-NOCONFIGURE=1 \
-./autogen.sh
-
-%configure \
-  --with-package-name='Fedora GStreamer package' \
-  --with-package-origin='http://download.fedoraproject.org' \
-  --enable-gtk-doc \
-  --enable-debug \
-  --disable-fatal-warnings \
-  --disable-silent-rules \
-  --disable-tests --disable-examples \
-  --with-ptp-helper-permissions=capabilities
-
-
-%make_build V=1
-
+%meson	\
+  -D package-name='Fedora GStreamer package' \
```

Comment:
do we want to say fedora here?

Reply:
I guess we can change that while we're here, you're right.